docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -89,7 +89,7 @@ node of the cluster:
 --------------------------------------------------
 POST _nodes/reload_secure_settings
 {
-  "reload_secure_settings": "s3cr3t"
+  "secure_settings_password": "s3cr3t"
 }
 --------------------------------------------------
 // NOTCONSOLE
```

Comment:
Because I forgot to enable them :) When this was first introduced, I think `NOTCONSOLE` was there to cater for the fact that we couldn't set the keystore password in a proper way in the docs cluster